Updated the Python Standalone 3.8 bundle from 3.8.5 to 3.8.11.
The Python 3.7 version configured is untouched as 3.7.9 is
already the latest version available.
